src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="/"
+      class="search wrap_form"
```

Comment:
You can add height styles to the search selector instead

```suggestion
      class="search"
```